### DISCUSSION TITLE
Gratitude

### DISCUSSION THREAD

**Discussion by wdfinch** [yarnpkg/berry#2737] [General]
Just wanted to take a moment to say thank you to everyone who has worked on this project. Yarn 2 is amazing and I love using it. I like how the project takes bold steps to fix problems and adds some really amazing features like constraints. I also really appreciate how quick the community is to help and reply to issues. The library does have a learning curve as there's a lot of new stuff, so the help is greatly appreciated. I've moved the entire frontend at my company to yarn 2 and the whole team loves it. Thank you and keep up the amazing work! 

Also special thanks to @merceyz and @arcanis. Aside from all the work you've put into building the library, you've been extremely helpful with answering questions and responding to issues. I really appreciate it.

**Comment by pk-nb** [comment]
Want to echo this—`yarn` has made a huge difference in what we are able to do with running larger projects and has made node development so much better across a variety of projects. Workspaces in particular have been a godsend for our teams. You are an inspiration.

Thank you so much!
